plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTableColumn.java

```diff
@@ -31,16 +31,19 @@
 public class CubridTableColumn extends GenericTableColumn
 {
     final static String [] customTypes = {"ENUM", "JSON"};
+    public boolean isForeignKey;
 
     public CubridTableColumn(
             @NotNull GenericTableBase table,
             @Nullable String columnName,
             @Nullable String dataType,
-            @Nullable Boolean autoIncrement,
+            @Nullable boolean autoIncrement,
+            @Nullable boolean isForeignKey,

```

Comment:
You may not annotate primitives with nullability annotations. They can't be null.
```suggestion
            boolean autoIncrement,
            boolean isForeignKey,
```